lib/formBuilder.tsx

```diff
@@ -308,7 +308,7 @@ const _getElementInitialValue = (element: FormElement, language: string): Respon
  * @param formRecord
  * @param language
  */
-const _getFormInitialValues = (formRecord: PublicFormRecord, language: string): Responses => {
+export const getFormInitialValues = (formRecord: PublicFormRecord, language: string): Responses => {
```

Comment:
There's a comment that references the old name here 

https://github.com/cds-snc/platform-forms-client/blob/06739009234cd6152f166df1f24b6496f653948e/lib/formBuilder.tsx#L273

Reply:
Updated